WalletWasabi.Gui/Tabs/WalletManager/HardwareWallets/ConnectHardwareWalletViewModel.cs

```diff
@@ -51,6 +52,13 @@ public ConnectHardwareWalletViewModel(WalletManagerViewModel owner) : base("Hard
 				.Subscribe(_ =>
 				{
 					SelectedWallet = Wallets.FirstOrDefault();
+
+					if (SelectedWallet is { })
+					{
+						Wallet wallet = WalletManager.GetWalletByName(SelectedWallet.WalletName);
```

Comment:
The name of the wallet here is something like Trezor_T. In the Wallet Explorer you have TrezorT_1. 
You have to investigate the code here and find the wallet file by fingerprint or xpub key.  
https://github.com/molnard/WalletWasabi/blob/31989092bac8a8221b95494637c9dd8394a84633/WalletWasabi.Gui/Tabs/WalletManager/HardwareWallets/ConnectHardwareWalletViewModel.cs#L316

That is why this is not an easy task.